### PR TITLE
Milestone 1:  use create-react-app to initialize a new React app

### PR DESCRIPTION
**Pull request by rajkumardas2701**
In this project, I have
- created app using "npm init react-app calculator"
- added prop-types and big.js using npm
- hosted app to heroku

### CODE REVIEW

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @rajkumardas2701! So far you have done well adding the necessary packages you will need for this project and deploying the app as required! :+1: However, there are a few issues for you to work on to complete this milestone. These include:

- [x]   Customize the README.md file such that it contains more relevant information about the project ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [x] Delete unnecessary files or assets, such as the React SVG Logo. Generally, the only mandatory file in the public folder is the `index.html` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/files-removed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by sumancrest0001** [comment]
### Status: changes required :gear:

**Hello @rajkumardas2701**, Thank you for making some changes in the project setup. There are some minor changes required to move to the next milestone.

**[Required]** Remove unnecessary files and images from the public directory as well.

**[Required]** Commit messages should be capitalized and in the present tense. 
For example, "Remove the SVG file" like this.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Please make the requested changes and submit for another code review. 
If you have any queries regarding this review, message me on Slack.**

**Suman Shrestha
Happy coding !!! :slightly_smiling_face:**

File: src/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+// import reportWebVitals from './reportWebVitals';
```
Comment (Forison) [inline src/index.js:5]: * [ ] Remove all commented code before the next review

File: src/index.js [hunk 2]
```diff
@@ -14,4 +14,4 @@ ReactDOM.render(
 // If you want to start measuring performance in your app, pass a function
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+// reportWebVitals();
```
Comment (Forison) [inline src/index.js:17]: * [ ] Remove all commented code before the next review

**Review by Forison — APPROVED** [review]
## Status::Approved✔️
#### Good job on addressing the issues raised earlier, you have now remove all unnecessary files.
## Kindly go ahead and merge the branch and proceed to the next milestone🤗.
#### Happy coding😎
Addo Forison

**Pull request merged**